examples/volume_plane_rendering.py

```diff
@@ -0,0 +1,230 @@
+"""
+Display one 3-D volume layer using the add_volume API and display it as a plane
```

Comment:
I wonder if in addition to this example you might want a really simple one just with a single slider that moves the plane through the volume at a fixed orientation - might be good to have that as a first example to show folks before the more complex widget

Reply:
done!

Reply:
love it!